tox.ini

```diff
@@ -50,9 +50,6 @@ commands =
         ansible-galaxy install -r {homedir}/openstack-ansible-role-requirements.yml; \
         {toxinidir}/scripts/linting-ansible.sh"
 
-[testenv:release-script]
-commands = python -m unittest discover -s scripts -p test_release.py -v
```

Comment:
Given that test_release.py is no longer used, it should be removed

Reply:
addressed